docs/guides/Integration-Guides/login-integration-guide/registering-credentials-to-set-up-apple-id-login.md

```diff
@@ -0,0 +1,145 @@
+
+1. Sign in to the [Apple Developer Portal](https://idmsa.apple.com/IDMSWebAuth/signin?appIdKey=891bd3417a7776362562d2197f89480a8547b108fd934911bcbea0110d07f757&path=%2Faccount%2F&rv=1).
+2. Click **Certificates, Identifiers & Profiles**, as shown below.
+
+   ![apple-credentials-1](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/images/apple-credentials-1-6-11.PNG)
+
+3. From the sidebar, click **Identifiers**, then click the blue add `+` icon.
+
+   ![apple-credentials-2](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/images/apple-credentials-2-7.PNG)
+
+4. Choose **App IDs** and click `Continue`:
+
+   ![apple-credentials-3](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/images/apple-credentials-3.PNG)
+
+5. Complete the fields to register the App ID:
+
+   - **Platform**: Select the app platform. You can choose between **iOS, tvOS, watchOS,** or **macOS**.
+   - **Description**: Write an internal description for the app. Don’t use special characters such as `@`, `&`, `*`, `'`, `"`.
+   - **App ID Prefix:** Non-editable field corresponding to your Apple Developer Team ID, which acts as a prefix to your App ID. Copy this Team ID and save it somewhere, as you will use it to configure the Apple ID login in the VTEX Admin later on. You can also find this Team ID on the [Apple Developer Portal](https://developer.apple.com/) top bar anytime you are logged in.
+   - **Bundle ID**: Choose between creating an **Explicit** App ID, used for a single app, or a **Wildcard**, used for a set of apps. Check [Apple’s documentation about App IDs](https://developer.apple.com/help/glossary/app-id/) for more details.
+
+      Then, define the Bundle ID, which works as your app’s unique identifier. It needs to be a uniform type identifier (UTI) string containing only alphanumeric characters (`A-Z`, `a-z`, `0-9`), hyphens (`-`), and/or periods (`.`). The string should be in reverse-DNS format, such as `com.domainname.appname`. Bundle IDs are case-sensitive.
+
+   ![apple-credentials-4](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/images/apple-credentials-4.PNG)
+
+6. Scroll down through the list of capabilities and check the box next to **Sign In with Apple**.
+
+   ![apple-credentials-5](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/images/apple-credentials-5.PNG)
+
+7. Click `Continue`, review the completed information, then click `Register`.
+
+The App ID created is the combination of the **App ID Prefix** (Team ID) and the **Bundle ID**. For example, `5PD8XQY9EW.com.mystore`.
+
+### Step 2: Creating a Services ID
+
+Next, to enable Sign in with Apple for each website, you must register a Services ID, verify your domain, and associate it to an app. The Services ID identifies the particular instance of your app and serves as the [OAuth client_id](https://www.oauth.com/oauth2-servers/client-registration/client-id-secret/).
+
+Go ahead and create a new identifier and choose Services IDs.
```

Comment:
Essa frase seria para introduzir o passo-a-passo abaixo? Achei um pouco confusa se eu deveria criar e depois seguir os passos ou se era uma intro para o passos. Se for uma intro., uma ideia seria deixar no mesmo parágrafo do texto anterior e usar ":" no final ou algo como "siga os passos abaixo..."